spec/rubocop/cop/factory_bot/factory_association_with_strategy_spec.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::FactoryBot::FactoryAssociationWithStrategy do
+  context 'when passing a `create` stategy' do
+    it 'flags the strategy' do
+      expect_offense(<<~RUBY)
+        factory :foo, class: 'FOOO' do
+          profile { create(:profile) }
+                    ^^^^^^^^^^^^^^^^ Use an implicit, explicit or inline definition instead of hard coding a strategy for setting association within factory.
+          profile { association :profile }
```

Comment:
```suggestion
```
This line is checked below.
However, it won't hurt to cover the case when the offensive one is not the only statement.